components/concourse-github-resource/assets/in

```diff
@@ -103,49 +92,51 @@ EOF
 } | tr -d '\n'
 }
 
-function approvals() {
-  local commit_sha=$1
-  local org=$2
-  local repo=$3
-  local token=$4
-  local api_response
-
-  api_response=$( \
-    curl --header 'Content-Type: application/json' \
-         --header "Authorization: bearer ${token}" \
-         --data "{ \"query\": \"$(build_query "${org}" "${repo}")\"}" \
-         "https://api.github.com/graphql" \
-  )
-
-  echo "${api_response}" \
-    | jq --raw-output .data.organization.repository.pullRequests.nodes[] \
-    | jq --raw-output ".reviews.nodes[] | select(.commit.oid==\"${commit_sha}\")" \
-    | jq --raw-output .author.login \
-    | sort -d
-}
+destination=$1
+cd "${destination}"
+head_ref=$(git rev-parse HEAD)
+
+api_response=$( \
+  curl --header 'Content-Type: application/json' \
+        --header "Authorization: bearer ${GITHUB_API_TOKEN}" \
```

Comment:
The original bearer token variable name was `bearer ${token}`, we should probably switch it back. :flushed:

Reply:
Done.